employees/serializers.py

```diff
@@ -50,3 +50,10 @@ class Meta:
             'description',
             'work_hours',
         )
+
+    def to_representation(self, instance):
+        data = super(ReportSerializer, self).to_representation(instance)
```

Comment:
As far as I know in python3 you don't have to write `super(Class, self)`. `super()` should be enough

Reply:
I've seen it around the web. Thought that maybe it's some sort of good practise, like explicit parametrising in function calls.

Reply:
Done